src/assets/styles/variables.scss

```diff
@@ -2,7 +2,10 @@ $red: #EF4134;
 $dark: #272F32;
 $blue: #537882;
 $light-blue: #9DBDC6;
+$lightest-blue: #D8E4E8;
```

Comment:
Grabbed a couple colors out of the design itself, these weren't in the styleguide